media/video-stitcher/updateCdnKeyAkamai.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2023, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(projectId, location, cdnKeyId, hostname, akamaiTokenKey) {
```

Comment:
Per the [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/#minimal-arguments), we should minimize the number of arguments our samples have.

Can we hard-code _some_ of these values? (e.g. `location`, `cdnKeyId`, and `hostname`?)

Reply:
Added some default values for most of the parameters.

Reply:
Ah - sorry for not being specific! I was referring to declaring these _as their own values_ (i.e. `const ... = ...`).

**Rationale:**
> Default arguments are better than those without defaults, but they're still more confusing to end-users than explicitly-declared `const`s.

Reply:
Moved some of these to explicitly-declared consts